bin/fastatic

```diff
@@ -26,6 +27,7 @@ fastatic({
 		logger.log(`Done. Optimised static files in: ${dest}`);
 	})
 	.catch(err => {
-		console.error(err.message);
+		errorReporter(err, program.logLevel);
```

Comment:
maybe errorReporter doesn't need to know about logLevel?
```javascript
.catch(err => {
  if (program.logLevel === 'verbose') {
    errorReporter(err);
    console.log(`Optimising failed. See above for details.`);
  }
  process.exitCode = 1;
})
```
And I think we can use the same logic for filesizeReporter. Results in simpler reporters and less dependencies.